jwst/datamodels/schemas/lev3_prod.schema.yaml

```diff
@@ -4,22 +4,62 @@ properties:
     type: object
     properties:
       resample:
+        # 2019-06-12: Title has changed to "Resampling parameter information" in keyword dictionary.
         title: Metadata describing resampling done using this data
         type: object
         properties:
           pointings:
+            # 2019-06-12: Title has changed to "Number of groups/pointings included in resampled product" in keyword dictionary.
             title: Number of pointings included in resampled product
             type: integer
             fits_keyword: NDRIZ
           product_exposure_time:
+            # 2019-06-12: Title has changed to "[s] Total exposure time for product" in keyword dictionary.
             title: Total exposure time for product
             type: number
             fits_keyword: TEXPTIME
           weight_type:
             title: Type of drizzle weighting to use in resampling input
             type: string
             enum: [exptime, error]
-            fits_keyword: WHT_TYPE
+            fits_keyword: RESWHT
+          drizzle_weight_scale:
```

Comment:
The schema editor tool put this and the following keywords into core.schema.yaml, but it seemed to me that they belonged here.  Let me know if I've got that wrong.

Reply:
This looks correct. @jdavies-st This won't break resample because it's using the data model, right?

Reply:
Correct.  It will just a diff in the regression tests.